src/mapping-v3-core.ts

```diff
@@ -240,6 +242,110 @@ export function handleMinterUpdated(event: MinterUpdated): void {
   contractEntity.save();
 }
 
+// Handle artist proposed address and splits updates
+// This is an event indicating that the artist has proposed a new set of
+// addresses and splits for a project.
+export function handleProposedArtistAddressesAndSplits(
+  event: ProposedArtistAddressesAndSplitsEvent
+): void {
+  // load associated project entity
+  const newEntityId = generateContractSpecificId(
+    event.address,
+    event.params._projectId
+  );
+  const project = Project.load(newEntityId);
+  if (!project) {
+    return;
+  }
+  // remove any existing proposed artist addresses and splits
+  const existingProposedArtistAddressesAndSplitsId =
+    project.proposedArtistAddressesAndSplits;
+  if (existingProposedArtistAddressesAndSplitsId !== null) {
+    project.proposedArtistAddressesAndSplits = null;
+    store.remove(
```

Comment:
@yoshiwarab actually, given my updated understanding of how overwriting works, do you think it would be okay to remove this entire `//remove any existing proposed artist addresses and splits` code block? When we create a new entity, it should just automatically overwrite any existing entity.

Reply:
Ah yeah that works even better! As long as all of the fields are overwritten (if any are not they'll keep the old values)

Reply:
SGTM! good call about all fields needing to be updated - we are good here because all fields will always be populated from the event. updated and pushed!